src/SprykerMiddleware/Zed/Process/Business/Process/Processor.php

```diff
@@ -160,13 +161,17 @@ protected function init(): void
         $this->processPlugin = $this->processPluginResolver
             ->getProcessConfigurationPluginByProcessName($this->processSettingsTransfer->getName());
 
-        $this->inputStream = $this->processPlugin
-            ->getInputStreamPlugin()
-            ->getInputStream($this->processSettingsTransfer->getInputPath());
+        $inputStreamPlugin = $this->processPlugin->getInputStreamPlugin();
+        if ($inputStreamPlugin instanceof OptionAwareStreamPluginInterface) {
+            $inputStreamPlugin->setOptions($this->processSettingsTransfer->getInputStreamOptions());
```

Comment:
It has a default value `{}` https://github.com/spryker-middleware/process/pull/52/files#diff-62a5899199d115dc6b3150e4fdac74c2R103. If it's empty and the is defined required option for specific Stream it will throw MissingOptionsException.

Reply:
You are not forced to use this code through the console.